app/src/main/java/com/edulectronics/tinycircuit/Views/Draggables/DeleteZone.java

```diff
@@ -0,0 +1,83 @@
+package com.edulectronics.tinycircuit.Views.Draggables;
+
+import android.content.Context;
+import android.graphics.Rect;
+import android.util.AttributeSet;
+import android.widget.ImageView;
+
+import com.edulectronics.tinycircuit.Controllers.CircuitController;
+import com.edulectronics.tinycircuit.Models.Components.Component;
+import com.edulectronics.tinycircuit.Views.CircuitActivity;
+import com.edulectronics.tinycircuit.Views.Draggables.Interfaces.DragSource;
+import com.edulectronics.tinycircuit.Views.Draggables.Interfaces.DropTarget;
+
+
+public class DeleteZone extends ImageView
+        implements DropTarget
+{
+    private DragController mDragController;
+
+    // Constructors
+    public DeleteZone (Context context) {
+        super  (context);
+    }
+    public DeleteZone (Context context, AttributeSet attrs) {
+        super (context, attrs);
+    }
+    public DeleteZone (Context context, AttributeSet attrs, int style)
+    {
+        super (context, attrs, style);
+    }
```

Comment:
In de hele klasse trouwens. Accolades horen op dezelfde regel als de methode declaratie.